ansible/library/interface_facts.py

```diff
@@ -12,10 +12,15 @@
 import datetime
 import getpass
 import pwd
-import ConfigParser
-import StringIO
 import json
 
+if sys.version_info < (3, 0):
+    import ConfigParser
+    import StringIO
```

Comment:
Are they not used? LGTM also alerts on them.

Reply:
They do not appear to be used. I've removed both the Python 2 and 3 imports.